f2/src/forms/AnonymousInfoForm.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useLingui } from '@lingui/react'
+import { Trans } from '@lingui/macro'
+import { Form } from 'react-final-form'
+import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
+import { RadioAdapter } from '../components/radio'
+import { Stack } from '@chakra-ui/core'
+import { useStateValue } from '../utils/state'
+import { FormArrayControl } from '../components/FormArrayControl'
+
+const validate = () => {
+  return {}
+}
+const clearData = dataOrig => {
+  let data = JSON.parse(JSON.stringify(dataOrig))
+  return data
+}
+
+export const AnonymousInfoForm = props => {
+  const { i18n } = useLingui()
+
+  const [data] = useStateValue()
+  const anonymous = {
+    ifanonymous: '',
+    ...data.formData.anonymous,
+  }
+  //TODO: Move this form data to some sort of a schema file instead?
+
+  const recurrenceCheck = {
+    ifanonymous: [],
+    ...data.formData.timeline,
+  }
```

Comment:
You should use the `anonymous` const declared above to initialize data. Otherwise this

Reply:
fixed